certified-connectors/ConsenSys/apiProperties.json

```diff
@@ -0,0 +1,156 @@
+{
+  "properties": {
+    "iconBrandColor": "#FFFFFF",
```

Comment:
@thedarkjester this color is not allowed. You can change it to light grey #FEFEFE color if you want to keep it as close to white as possible. The icon background color should also match the color from iconBrandColor property. More information can be found [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector).